hack/remove-members.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var dryrun bool
+var configPath string
+
+func main() {
+	flag.StringVar(&configPath, "path", ".", "Path to config directory/subdirectory")
+	flag.BoolVar(&dryrun, "dryrun", true, "Enable Dryrun or not. This flag controls whether the changes are simulated or live. If the changes are simulated it will only print the removal details and the commit message")
+
+	flag.Parse()
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "Usage: remove-members [flags] <file-containing-members-list>\n")
+		flag.PrintDefaults()
+	}
+
+	if len(flag.Args()) != 1 {
+		flag.Usage()
+		os.Exit(0)
+	}
+
+	memberList, err := readMemberList(flag.Args()[0])
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err = removeMembers(memberList, configPath); err != nil {
+		log.Fatal(err)
+	}
+
+}
+
+//readMemberList reads the list of members to be removed from the given filepath
+func readMemberList(path string) ([]string, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	var members []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		members = append(members, scanner.Text())
+	}
+	return members, scanner.Err()
+}
```

Comment:
How about simplifying this using `ioutil.ReadFile` as used in `removeMemberFromFile`?

Note: `ioutil.ReadFile` has been deprecated in Go 1.16 although it shouldn't be a blocker since this repository still uses Go 1.13.